modules/core/src/test/java/org/apache/ignite/internal/processors/performancestatistics/AbstractPerformanceStatisticsTest.java

```diff
@@ -71,7 +71,7 @@ public static void cleanPerformanceStatisticsDir() throws Exception {
     }
 
     /** Starts collecting performance statistics. */
-    protected static void startCollectStatistics() throws Exception {
+    public static void startCollectStatistics() throws Exception {
```

Comment:
why do you change protected here ?

Reply:
Because I use this method in `RemoveConstantsFromQueryTest` to check that performance statistics will store queries without sensitive data